server/test/services/applicant/BlockTest.java

```diff
@@ -346,6 +346,27 @@ public void isAnswered_outsideChangesToApplicantData_updatesCompletionCheck() {
     assertThat(block.isAnsweredWithoutErrors()).isTrue();
   }
 
+  @Test
+  public void answeredByUserQuestionsCount_includesStatic() {
```

Comment:
question (non-blocking):  Shouldn't this say "excludesStatic" since we're excluding static questions from the count?